doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1533,18 +1581,8 @@ Response code:
             ],
             "scopes": [
                 {
-                    "type": "FIWARE::Filter::qString",
-                    "value": "temperature>40;colour=black,red;+status"
-                },
-                {
-                    "type": "FIWARE::Filter::Location",
-                    "value": {
-                        "circle": {
-                            "centerLatitude": "40.418889",
-                            "centerLongitude": "-3.691944",
-                            "radius": "13500"
-                        }
-                    }
+                    "type": "FIWARE::...",
```

Comment:
In addition, include a note in the text explaining that scopes are under definition (as we have donde in similar cases).